### PR TITLE
re-enable microscope in CI

### LINKED ISSUE #4011: CI: re-enable microscope in K8s CI
Due to issues discovered in #3994, running microscope in the K8s CI has been disabled (which was done by #4012). This tracks the re-enabling of microscope in K8s CI.

### PR DESCRIPTION
**Pull request by nebril**
fixes #4011

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by eloycoto** [comment]
should not this happens when the monitor command is closed?

**Comment by nebril** [comment]
This is not enough yet, seems like our helpers send SIGHUP to spawned process, microscope process doesn't handle this properly, I will add support for that in microscope repo.

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by aanm** [comment]
@nebril if flake please file GH issue

**Comment by nebril** [comment]
@aanm I think this flake was fixed in master already, let me check again

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -344,47 +344,42 @@ func (kub *Kubectl) Logs(namespace string, pod string) *CmdRes {
...
-			if err != nil {
-				log.WithError(err).Errorf("cannot create monitor log file")
-			}
-			return nil
+		err = ioutil.WriteFile(
+			filepath.Join(testPath, monitorLogFileName),
+			res.CombineOutput().Bytes(),
+			LogPerm)
+		if err != nil {
+			log.WithError(err).Errorf("cannot create monitor log file")
```
Comment (eloycoto) [inline test/helpers/kubectl.go:378]: Ouch, could you add a return err here please?

**Review by eloycoto — APPROVED** [review]
Small nitpick from the past, but if you can fix it'll be great ;-)

**Comment by nebril** [comment]
test-me-please

File: test/helpers/kubectl.go (hunk 1 above)
Comment (nebril) [reply test/helpers/kubectl.go:378]: done

Comment (eloycoto) [reply test/helpers/kubectl.go:378]: Thanks!

**Comment by ianvernon** [comment]
Given that the build failed, marking as wip.

**Comment by nebril** [comment]
test-me-please

**Pull request merged**